sql-queries/delete-non-citation-relation-types.sql

```diff
@@ -1,6 +1,29 @@
 BEGIN;
+
+-- Delete non-citation relation types from assertions
 DELETE FROM public.assertions
-WHERE (relation_type_id is not null
-AND relation_type_id not in ('cites', 'is-cited-by', 'references', 'is-referenced-by', 'is-supplemented-by', 'is-supplement-to'))
+WHERE relation_type_id is not null
+    AND source_id = '3644e65a-1696-4cdf-9868-64e7539598d2'
```

Comment:
@lizkrznarich are we only removing CZI duplicates or all sources?

Reply:
@kaysiz CZI assertions don't have relation types, so it's just datacite assertions that are being deleted here. because of that, it doesn't matter if source id is included in the query or not, but 3644e65a-1696-4cdf-9868-64e7539598d2 is the correct source id (datacite) to use in this query.